test/parallel/test-buffer-of-no-deprecation.js

```diff
@@ -0,0 +1,8 @@
+// Flags: --pending-deprecation --no-warnings
+'use strict';
+
+const common = require('../common');
+
+process.on('warning', common.mustNotCall());
```

Comment:
Should the `--no-warnings` flag be dropped?

Reply:
No, the `--no-warnings` just stops the console output. The `process.on('warning')` event will still emit.